ocfweb/context_processors.py

```diff
@@ -3,13 +3,18 @@
 
 from ocflib.lab.hours import DayHours
 
+from ocfweb.component.banner import get_banner_message
+
 
 def ocf_template_processor(request):
     now = datetime.now()
     today = date.today()
     hours = DayHours.from_date(today)
 
+    banner = get_banner_message()
+
     return {
         'lab_is_open': hours.is_open(now),
         'current_lab_hours': hours,
+        'force_lab_closed': banner.visible and banner.force_lab_closed,
```

Comment:
Well you can still do that, but you're right they could be independent. I
was thinking about returning everything to normal just by changing
`visible` to false. If they were independent, where would we specify to
force close the lab?
On Sep 26, 2015 12:35 AM, "Chris Kuehl" <notifications@github.com> wrote:

> In ocfweb/context_processors.py
> <https://github.com/ocf/ocfweb/pull/34#discussion_r40491182>:
>
> >      return {
> >          'lab_is_open': hours.is_open(now),
> >          'current_lab_hours': hours,
> > +        'force_lab_closed': banner.visible and banner.force_lab_closed,
>
> I think we shouldn't tie force_lab_closed and the banner message together.
>
> For example we might want to show a banner "Printing is down" without
> closing the entire lab. They can probably just be independent.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ocf/ocfweb/pull/34/files#r40491182>.
>